docs/datadog_dashboard.md

```diff
@@ -0,0 +1,135 @@
+# Datadog Dashboards
+This feature is in preview.
+
+## Overview
+The DatadogDashboard Custom Resource Definition (CRD) allows users to create [dashboards][1] using the Operator and manage them as Kubernetes resources.
+
+## Prerequisites
+
+- Datadog Operator v1.9+
+- **[Helm][2]** for deploying the Datadog Operator
+- **[`kubectl` CLI][3]** for installing a `DatadogDashboard`
+
+Tests were performed on Kubernetes versions >= `v1.30.0`
+
+## Adding a DatadogDashboard
+
+To deploy a `DatadogDashboard` with the Datadog Operator, use the [`datadog-operator` Helm chart][3].
+
+1. Install the [Datadog Operator][4]:
+
+   First, add the Datadog Helm chart with
+
+    ```shell
+    helm repo add datadog https://helm.datadoghq.com
+    ```
+
+    1. Run the following command, substituting your [Datadog API and application keys][5]:
```

Comment:
[5] takes you to https://app.datadoghq.com/dashboard/lists not the API/APP keys or public doc were to find those